### DISCUSSION TITLE
Tina ignoring path and creating new one

### DISCUSSION THREAD

**Discussion by Inigojv** [tinacms/tinacms#5214] [Help] [answered]
Hi everyone, 

I am developing a website using Astro and struggling to get TinaCMS correctly set up.

In the config file the path is set to "src/content/post", however when I create a new post in TinaCMS it creates a new folder "content/post".

I have tried removing src but it still creates this folder.

Does anyone know why tina is interpreting the path in this way?

### This is the path it is creating

content/post/test.md

###  This is the path I want

src/content/post/test.md

**Comment by kldavis4** [comment]
Post a reproduction please

**Reply by Inigojv** [reply]
Thank you for your response, I have updated the original post.

Sorry if that is not enough information, I'm quite new to github/ coding in general.

**Reply by kldavis4** [reply]
Post your actual repo or a repo with a reproduction

**Reply by Inigojv** [reply]
Thank you for your patience! I have created a repo as a reproduction here: https://github.com/Inigojv/website

**Comment by kldavis4** [comment, marked as answer]
It looks like your tina-lock.json doesn't match your tina config. 

If you look at https://github.com/Inigojv/website/blob/main/tina/tina-lock.json, you can see the path in the posts collection is configured to `content/posts` and not `src/content/posts`.

Anytime you update the tina/config.ts, you need to run the dev or build command locally to update the tina/tina-lock.json and then commit the change and push it to your github repository. Tina Cloud relies on the tina-lock.json in your github repository for its schema.

The only thing strange here is that there is a check that runs to validate that the tina cloud schema matches the local version when running build and it looks like that isn't checking that the path matches, so we probably need to fix that. @wicksipedia @ncn-ssw

**Reply by ncn-ssw** [reply]
I had a quick look at the relevant code paths, but I could not devise a reasonable theory as to why the different paths were not being detected.

I have created an issue at https://github.com/tinacms/tinacms/issues/5221 for further investigation.
